labs-ios-starter/Application/Network/NetworkService.swift

```diff
@@ -16,7 +16,7 @@ typealias URLCompletion = ((Result<Data, ErrorHandler.NetworkError>) -> Void)
 /// Completion Handler with String .success and NetworkError .failure
 typealias CompleteWithString = (Result<String, ErrorHandler.NetworkError>) -> Void
 /// Completion Handler with [Topic] .success and NetworkError .failure
-typealias CompleteWithTopics = (Result<[Topic], ErrorHandler.NetworkError>) -> Void
+typealias CompleteWithTopics = (Result<[Topic]?, ErrorHandler.NetworkError>) -> Void
```

Comment:
The idea of the Result type is to guarantee a value on success and error on failure. If you need to unwrap something, that's ok - if the unwrapping fails, you can complete with error and return from the method
```suggestion
typealias CompleteWithTopics = (Result<[Topic], ErrorHandler.NetworkError>) -> Void
```

Reply:
After reviewing the change that prompted this, You can replace `[Topic]` with `Void` here. If the completion handler is being used elsewhere, let's create a new CompletionHandler, or I believe there's an existing one with Void as the success Type

Reply:
Good call